.github/workflows/expectfail.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 if $1; then
-    echo "FALSE POSITIVE! Expected failure but succeeded"
-    exit 1
+    echo "Success"
```

Comment:
You could remove reference to this .sh from the yml, and remove this file itself.
Now the workflow will pass even on failure, since the result of this shell script will be the result of the last echo